storageapi/src/vespa/storageapi/mbusprot/protobuf/common.proto

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+syntax = "proto3";
+
+option cc_enable_arenas = true;
+
+package storage.mbusprot.protobuf;
+
+// Note: we use a *Request/*Response naming convention rather than *Command/*Reply,
+// as the former is the gRPC convention and that's where we intend to move.
+
+// Next tag to use: 2
+message BucketSpace {
+   uint64 space_id = 1;
+}
+
+// Next tag to use: 2
+message BucketId {
+   fixed64 raw_id = 1;
+}
+
+// Next tag to use: 3
+message Bucket {
+    uint64  space_id      = 1;
+    fixed64 raw_bucket_id = 2;
+}
+
+// Next tag to use: 9
+message BucketInfoV1 {
```

Comment:
Re discussion:
- Remove BucketInfoV2 and BucketInfo
- Rename BucketInfoV1 to BucketInfo
- Rename checksum to legacy_checksum
- Create message for new checksum and put into BucketInfo

Reply:
Done, although I haven't added a new message for the V2 checksum since it's not yet fully spec'd. We can now finally take that kind of liberty since adding message fields is painless! 🙂🦄